services/core-api/app/api/services/NRIS_API_service.py

```diff
@@ -61,7 +62,7 @@ def _process_NRIS_data(raw_data):
         'last_inspector': None,
         'num_open_orders': 0,
         'num_overdue_orders': 0,
-        #'section_35_orders': 0, no aggregate, FE filters will show where violation = 35
+                                                                          #'section_35_orders': 0, no aggregate, FE filters will show where violation = 35
```

Comment:
odd formatting

Reply:
This is the the yapf formatting we can change the config but that is the way it is configured to format comments.